web/src/api.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Listing} from 'interfaces';
+
+/**
+ * Fetches all Listings.
+ */
+export const getAllListings = async (): Promise<Listing[]> => {
+  const res = await fetch(`${process.env.REACT_APP_BACKEND_URL}/listings`);
+  return await res.json();
```

Comment:
Here can remove the `await` also. 
```suggestion
  return res.json();
```

Reply:
oh ya thx for reminding